NeuroID/Utilities/NetworkMonitoringService.swift

```diff
@@ -18,6 +18,8 @@ internal class NetworkMonitoringService {
 
     internal private(set) var isConnected: Bool = false
     internal private(set) var connectionType: ConnectionType = .unknown
+    
+    internal static var connectionTypeAccessor: ConnectionType = .unknown
```

Comment:
Similar to the above comment, why are we storing the same property twice when we create a class instance?